src/main/java/org/motechproject/newebodac/service/RoleService.java

```
@@ -57,18 +58,28 @@ public RoleDto create(RoleDto roleDto) {
   public RoleDto update(UUID id, RoleDto roleDto) {
     UserRole role = roleRepository.findById(id).orElseThrow(() ->
         new EntityNotFoundException("Role with id: {0} not found", id.toString()));
-    MAPPER.update(roleDto, role);
+    if (role.getName().equals(ADMIN_ROLE)) {
```

I think it will be better to throw an exception if user tries to change the admin role